matter_server/client/client.py

```diff
@@ -53,16 +41,18 @@ class MatterClient:
 
     def __init__(self, ws_server_url: str, aiohttp_session: ClientSession):
         """Initialize the Client class."""
-        self.ws_server_url = ws_server_url
```

Comment:
Removing the server url is a breaking change. I think it's a good change, but we should call it out in the PR description and label the PR as breaking, in that case.

Reply:
Are we using it in the HA integration ? If so I can expose it as a property.
If not, we should not care at this point imo.

Reply:
I'm not sure. Regardless we should document the changes in the release notes. Labeling it here makes that easier.